bluetooth/src/commonTest/kotlin/BluetoothScanTest.kt

```diff
@@ -26,9 +27,11 @@ import kotlin.test.assertTrue
 
 class BluetoothScanTest: BluetoothFlowTest<Boolean>() {
 
-    override val flow: suspend () -> Flow<Boolean> = {
-        setup(Setup.BLUETOOTH)
-        bluetooth.isScanning()
+    override val flow: () -> Flow<Boolean> = {
+        runBlocking {
```

Comment:
Maybe it would be better to re-add suspend on the declaration of this `abstract val`.

Reply:
done